src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -663,6 +666,10 @@ public ConsumerDTO getConsumer(@Verify(Consumer.class) String uuid) {
                     consumer = this.regenerateIdentityCertificate(consumer);
                 }
             }
+            else if (!this.hypervisorType.isHypervisor(consumer.getTypeId())) {
+                log.info("No identity certificate found for consumer: {}. Generating a new one", uuid);
+                consumer = this.regenerateIdentityCertificate(consumer);
+            }
```

Comment:
@Januson Based on our previous discussions, we should revert this. We don't want to regenerate the identity cert if there isn't one, because that would mean they either don't need one (it's a hypervisor), or it was expired and the CertCleanupJob deleted it (at which point, they should just be re-registering manually with username/password)

Reply:
Done.